backend/chainlit/cli/__init__.py

```diff
@@ -160,3 +164,13 @@ def chainlit_hello(args=None, **kwargs):
 def chainlit_init(args=None, **kwargs):
     trace_event("chainlit init")
     init_config(log=True)
+
+
+@cli.command("create-secret")
+@click.argument("args", nargs=-1)
+def chainlit_create_secret(args=None, **kwargs):
+    trace_event("chainlit secret")
+
+    print(
+        f"Copy the following secret into your .env file:\nCHAINLIT_JWT_SECRET={random_secret()}"
```

Comment:
maybe add a warning that if it is lost users will not be able to authenticate anymore